_site/CNAME

```diff
@@ -0,0 +1 @@
+kitsunekeyboards.com
```

Comment:
Also `_site` needs to be in `.gitignore` pretty please :)

Reply:
see if I got them in the right place ... think I did it 😁